### PR TITLE
Wip libraryitem

### PR DESCRIPTION
**Pull request by cvrkota**
Adding ability to add resource to the database, to view one resource at the time, and to view all resources from the library

### CODE REVIEW

File: db/migrate/20200917005322_create_items.rb
Comment (spinecone) [inline db/migrate/20200917005322_create_items.rb:4]: Are there any other fields we might want on this table? Thinking about things like the type of media (is it a youtube video, library book, online article, etc.), the url if it's an online resource, the date it was made?

File: db/schema.rb
Comment (spinecone) [inline db/schema.rb:16]: What is the "new" field going to be used for?

File: app/models/item.rb
Comment (spinecone) [inline app/models/item.rb:2]: Validations for maximum length on these fields could also be useful.

File: app/controllers/items_controller.rb
Comment (spinecone) [inline app/controllers/items_controller.rb:19]: Typically after creating something, we redirect to the "show" page for that object. If we'd rather go back to the "new" form instead that's fine too, just wanted to mention it as something that might feel more intuitive for the user.

File: app/controllers/items_controller.rb
Comment (spinecone) [inline app/controllers/items_controller.rb:27]: Indenting got a little weird here!

File: app/helpers/items_helper.rb
Comment (spinecone) [inline app/helpers/items_helper.rb:1]: I would recommend deleting this file if we don't currently have a use for it.

**Comment by spinecone** [comment]
Let's merge this in after:
- Address removing the "new" field, fixing indenting, and deleting the ItemsHelper
- Wait for the admin login PR to be merged
- Deal with merge conflicts here that will come from that 😭 
- We get a 👍 from one of the reviewers

File: app/views/items/index.html.erb
Comment (nostango) [inline app/views/items/index.html.erb:9]: There is a warning about the file not end in a newline

File: config/routes.rb
Comment (nostango) [inline config/routes.rb:3]: I'm curious to how the routes file with the Devise login will work with this one as we have a main page and a separate database as well.

**Review by niraulap — COMMENTED** [review]
Went through everything! Did not catch anything specific that had not been already caught!

File: test/controllers/items_controller_test.rb
Comment (athaichi) [inline test/controllers/items_controller_test.rb:8]: should you check to make sure that you can access items that have already been added to your database?

**Review by athaichi — DISMISSED** [review]

**Review by nostango — APPROVED** [review]

**Pull request merged**
